src/lang.ts

```diff
@@ -171,9 +168,12 @@ export function getPropertyDescriptor(object: Object, property: string): Propert
 }
 
 export function isIdentical(a: any, b: any): boolean {
```

Comment:
Since you seem to have moved the logic from `object.is` here, should we also consider what to do with that module?  It seems like it will be removable once we also create `assign` and `deepAssign` in `lang`.

Meanwhile, I'm actually not sure whether the `-0` code is even necessary; `-0 === +0` is already `true` in JS.